measurecrack.py

```diff
@@ -0,0 +1,175 @@
+import cv2, imutils
+import numpy as np
+
+# Measures crack using raio method
+def measure_crack(image):
+    #Get dimensions of blue rectangle
+    width, height = blue_rectangle(image)
+    
+    # Return larger dimension. Multiply ratio by 1.85
+    if height > width:
+        return (height/width)*1.9
```

Comment:
please dont use magic numbers.  pull this hardcoded 1.9 out to a constant and give it a name.

also, you say to multiply the ratio above by 1.85 and you multiply by 1.9. thats confusing.  also when you change this to a constant change the comment to reflect that value.  you should describe what that 1.85 means and why youre using it